From ecashin@coraid.com Wed Sep 20 11:53:04 2006
Message-ID: <2d549e40b0edb620a49520e40aa02bb8@coraid.com>
Date: Wed, 20 Sep 2006 14:34:41 -0400
To: linux-kernel@vger.kernel.org
Cc: ecashin@coraid.com, Greg K-H <greg@kroah.com>
Subject: aoe: eliminate isbusy message
From: "Ed L. Cashin" <ecashin@coraid.com>

This message doesn't help users because the circumstance isn't problematic.

Signed-off-by: "Ed L. Cashin" <ecashin@coraid.com>
Acked-by: Alan Cox <alan@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/block/aoe/aoedev.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- gregkh-2.6.orig/drivers/block/aoe/aoedev.c
+++ gregkh-2.6/drivers/block/aoe/aoedev.c
@@ -20,11 +20,8 @@ aoedev_isbusy(struct aoedev *d)
 	f = d->frames;
 	e = f + d->nframes;
 	do {
-		if (f->tag != FREETAG) {
-			printk(KERN_DEBUG "aoe: %ld.%ld isbusy\n",
-				d->aoemajor, d->aoeminor);
+		if (f->tag != FREETAG)
 			return 1;
-		}
 	} while (++f < e);
 
 	return 0;
